Library/Formula/vramsteg.rb

```diff
@@ -1,8 +1,9 @@
 class Vramsteg < Formula
   desc "Add progress bars to command-line applications"
   homepage "https://tasktools.org/projects/vramsteg.html"
-  url "https://taskwarrior.org/download/vramsteg-1.0.1.tar.gz"
-  sha256 "bc47e078079a845fa9c9cc5e4c9f4585402430ac6efc82ea6ff607506af8bdb9"
+  url "https://tasktools.org/download/vramsteg-1.1.0.tar.gz"
+  sha256 "9cc82eb195e4673d9ee6151373746bd22513033e96411ffc1d250920801f7037"
+  head "https://git.tasktools.org/scm/ut/vramsteg.git", :branch => "1.1.1", :shallow => false
```

Comment:
Mindless copying and pasting. I was basing this formula off another that hit the same bitbucket server and assumed it was necessary. I just tested it and it works fine. Thanks.